example/all.yaml

```diff
@@ -144,6 +144,7 @@ linter:
     - prefer_void_to_null
     - provide_deprecation_message
     - public_member_api_docs
+    - pubspec_no_http_urls
```

Comment:
Should we consider calling it `avoid_http_urls` or `avoid_http_links` ?

I imagine that we should also extend it to cover HTTP links to git dependencies, and hosted-urls using HTTP (unless they are localhost)

Reply:
We have generally been trying to move away from terms like "avoid" and "prefer" because the firmness of the advice tends to change over time.

I could perhaps suggest an alternative if I better understood what we're trying to avoid and why. Is there an issue discussing this lint? The 'details' suggests that this is to catch potentially unsecured access. If that's correct then consider a name that reflects the security concern, such as 'only_secure_dependencies'.

Reply:
I created an issue now: https://github.com/dart-lang/pub/issues/3211

Reply:
A name like `only_secure_dependencies` might be better too. Since we probably should discourage `git://` dependencies.

Reply:
I like this suggestion.  Maybe we should be slightly more explicit though.  How about `only_secure_pub_dependencies`?